packages/react-app/src/views/Room.jsx

```diff
@@ -100,6 +104,7 @@ export default function Rooms({
 
       // notify user of signIn
       setIsSignedIn(true);
+      setConfetti(true);
```

Comment:
I think we need the confetti to show once distribution has been confirmed on chain